test/shared/createElement.test.js

```diff
@@ -32,9 +32,10 @@ describe('createElement(jsx)', () => {
 		expect(<Test />).to.have.property('type', Test);
 	});
 
-	it('should set VNode._self property to prevent json injection', () => {
+	it('should set VNode.constructor property to prevent json injection', () => {
 		const vnode = <span />;
-		expect(vnode._self).to.equal(vnode);
+		// eslint-disable-next-line no-proto
```

Comment:
Guess this one is a left over from the proto approach?

Reply:
Good spot, thanks 👍